operator/src/main/java/oracle/kubernetes/operator/builders/WatchBuilder.java

```diff
@@ -29,7 +29,7 @@
   /** Ignored for watches. */
   private static final String START_LIST = null;
 
-  private static final Boolean ALLOW_BOOKMARKS = false;
+  private static final Boolean ALLOW_BOOKMARKS = true;
```

Comment:
The ALLOW_BOOKMARKS is set to `true` in WatchBuilder but I don't see BOOKMARK watch event handled in Watcher.java. My understanding of Watch bookmark feature is that it will send an event of type BOOKMARK to mark that all changes up to a given resourceVersion the client is requesting have already been sent. I was imagining that we'll update the resource version in Watcher.java with resource version set in BOOKMARK event . https://kubernetes.io/docs/reference/using-api/api-concepts/#watch-bookmarks

Reply:
I found that the behavior of the default branch of the switch was identical to what we would do for a specific BOOKMARK event, so there was no need to make further code change.